bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/thw_5f_ckm_LO_ctcvcp_MH/thw_5f_ckm_LO_ctcvcp_MH_proc_card.dat

```diff
@@ -0,0 +1,18 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
```

Comment:
these datacards need a little bit of cleanup. there are multiple import model definitions (only the last one matters) and proton and jet definitions should be removed, and also unused lepton and neutrino definitions, cfr https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T
can you open another PR to fix them please